tests/Api/Responses/admin/payment_method/create_payment_method_response.json

```
@@ -10,9 +10,8 @@
     "enabled": true,
     "translations": {
         "en_US": {
-            "@id": "\/api\/v2\/admin\/payment-method-translations\/@integer@",
+            "@id": "\/api\/v2\/admin\/payment-method\/@string@\/translations\/@string@",
```

```suggestion
            "@id": "\/api\/v2\/admin\/payment-method\/test\/translations\/en_US",
```
These values are not autogenerated, we can hardcode them